tests/dummy/app/styles/app.css

```diff
@@ -187,3 +187,13 @@ body {
   background-color: black;
   opacity: .4;
 }
+
+.virtual-each {
```

Comment:
Is there any reason we don't just add this as a pull request to `virtual-each` itself? If every app that uses `virtual-each` needs this CSS to function, then it feels like this should be gotten for free, merely by including `virtual-each` in your project